### DISCUSSION TITLE
Run android apps on crankshaft

### DISCUSSION THREAD

**Discussion by Basline-LineTech** [opencardev/crankshaft#574] [Q&A]
Is there any way to install android apps directly on Crankshaft or port them to android auto?

Ps: I'm really newbie on running android on Linux so I might need an step by step
PS2: The reason I want it is because it would be really nice to have my country driver's license app directly on crankshaft

**Comment by GaupeLnx** [comment]
Android Auto is not android, apps are not running on AA but on your phone and mirrored  to the AA screen. 
So if you driver's s license app is not android auto compatible it won't mirror to the AA screen.
